### ISSUE TITLE
Uncaught TypeErrors

### ISSUE THREAD

**Issue by backjoob** [mdcurrie/dominion#26]
5index.js:48 Uncaught TypeError: socket.send is not a function
    at index.js:48
(anonymous) @ index.js:48
setInterval (async)
(anonymous) @ index.js:47
Ui @ react-dom.production.min.js:198
lk @ react-dom.production.min.js:255
./node_modules/scheduler/cjs/scheduler.production.min.js.exports.unstable_runWithPriority @ scheduler.production.min.js:18
fg @ react-dom.production.min.js:120
Sj @ react-dom.production.min.js:254
(anonymous) @ react-dom.production.min.js:254
X @ scheduler.production.min.js:17
./node_modules/scheduler/cjs/scheduler.production.min.js.H.port1.onmessage @ scheduler.production.min.js:14

**Comment by RasPat1** [comment]
#28 Should fix it and then some additional changes here... for fun! #29

**Comment by mdcurrie** [comment]
stack trace in heroku after merging #28. i see this error after logging in with two users

```
2021-02-03T21:10:49.216371+00:00 heroku[router]: sock=backend at=error code=H18 desc="Server Request Interrupted" method=GET path="/__webpack_hmr" host=marcusdominion2.herokuapp.com request_id=9e780faf-ee3d-4cf6-bdd1-5bf37776179b fwd="163.114.132.7" dyno=web.1 connect=1ms service=1849ms status=503 bytes= protocol=http
2021-02-03T21:10:49.200527+00:00 app[web.1]: /app/node_modules/redux/lib/redux.js:210
2021-02-03T21:10:49.200538+00:00 app[web.1]:       throw new Error('Actions may not have an undefined "type" property. ' + 'Have you misspelled a constant?');
2021-02-03T21:10:49.200539+00:00 app[web.1]:       ^
2021-02-03T21:10:49.200539+00:00 app[web.1]: 
2021-02-03T21:10:49.200540+00:00 app[web.1]: Error: Actions may not have an undefined "type" property. Have you misspelled a constant?
2021-02-03T21:10:49.200541+00:00 app[web.1]:     at dispatch (/app/node_modules/redux/lib/redux.js:210:13)
2021-02-03T21:10:49.200541+00:00 app[web.1]:     at Object.dispatch (/app/node_modules/@redux-saga/core/dist/redux-saga-core.prod.cjs.js:1321:22)
2021-02-03T21:10:49.200542+00:00 app[web.1]:     at WebSocket.<anonymous> (/app/dist/server.js:1:47791)
2021-02-03T21:10:49.200542+00:00 app[web.1]:     at WebSocket.emit (events.js:315:20)
2021-02-03T21:10:49.200542+00:00 app[web.1]:     at Receiver.receiverOnMessage (/app/node_modules/express-ws/node_modules/ws/lib/websocket.js:720:20)
2021-02-03T21:10:49.200543+00:00 app[web.1]:     at Receiver.emit (events.js:315:20)
2021-02-03T21:10:49.200543+00:00 app[web.1]:     at Receiver.dataMessage (/app/node_modules/express-ws/node_modules/ws/lib/receiver.js:414:14)
2021-02-03T21:10:49.200544+00:00 app[web.1]:     at Receiver.getData (/app/node_modules/express-ws/node_modules/ws/lib/receiver.js:346:17)
2021-02-03T21:10:49.200544+00:00 app[web.1]:     at Receiver.startLoop (/app/node_modules/express-ws/node_modules/ws/lib/receiver.js:133:22)
2021-02-03T21:10:49.200545+00:00 app[web.1]:     at Receiver._write (/app/node_modules/express-ws/node_modules/ws/lib/receiver.js:69:10)
2021-02-03T21:10:49.223555+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2021-02-03T21:10:49.224131+00:00 app[web.1]: npm ERR! errno 1
2021-02-03T21:10:49.225630+00:00 app[web.1]: npm ERR! dominion@1.0.0 start: `node ./dist/server.js`
2021-02-03T21:10:49.225828+00:00 app[web.1]: npm ERR! Exit status 1
2021-02-03T21:10:49.226082+00:00 app[web.1]: npm ERR! 
2021-02-03T21:10:49.226295+00:00 app[web.1]: npm ERR! Failed at the dominion@1.0.0 start script.
2021-02-03T21:10:49.226526+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2021-02-03T21:10:49.218154+00:00 heroku[router]: at=info method=GET path="/dominion?username=hellomto" host=marcusdominion2.herokuapp.com request_id=0195cec5-e444-4c7f-8087-6bcdf9d4db5d fwd="163.114.132.7" dyno=web.1 connect=4ms service=5780ms status=101 bytes=129 protocol=http
2021-02-03T21:10:49.546996+00:00 app[web.1]: 
2021-02-03T21:10:49.547266+00:00 app[web.1]: npm ERR! A complete log of this run can be found in:
2021-02-03T21:10:49.547365+00:00 app[web.1]: npm ERR!     /app/.npm/_logs/2021-02-03T21_10_49_228Z-debug.log
2021-02-03T21:10:49.614906+00:00 heroku[web.1]: Process exited with status 1
2021-02-03T21:10:49.661701+00:00 heroku[web.1]: State changed from up to crashed
```

**Comment by RasPat1** [comment]
Fixed with #31 .

Also merge #29 plz
